RELEASING.md

```diff
@@ -158,6 +158,7 @@ Tagging the Release
    $ git push upstream v$MAJOR.$MINOR.$PATCH
    $ git push upstream v$MAJOR.$MINOR.x
    ```
+6. Close the release milestone.
 
```

Comment:
Maybe add a "Create a new Milestone" in the branching section?

Reply:
Good idea. It was semi-implicit before. Done.